Makefile

```diff
@@ -324,16 +328,79 @@ copr: upload-srpms ## build in copr
 	./misc/copr-build.py https://$(SERVER)/$(REMOTE_PATH)/SRPMS/$(SRPM_BASE)
 
 #
-#	deb build
+#	release
 #
 
-deb: ## build debian package
-	./misc/gen-deb-changelog-from-git.sh
-	dpkg-buildpackage
-	# especially when building in Docker container, pull build artifact in project directory.
-	cp ../mgmt_*_amd64.deb ./
-	# cleanup
-	rm -rf debian/mgmt/
+release: releases/mgmt-$(VERSION)-release.url
+
+releases/mgmt-$(VERSION)-release.url: $(DEB_TAR) $(RPM_TAR) $(PACMAN_TAR)
+	@echo "Creating github release..."
+	hub release create \
+		-F <( echo -e "$(VERSION)\n";echo "License: GPLv3" ) \
+		-a $(DEB_TAR) \
+		-a $(RPM_TAR) \
+		-a $(PACMAN_TAR) \
+		$(VERSION) \
+		> releases/mgmt-$(VERSION)-release.url
```

Comment:
I think if you're going this method, you need an `|| rm -f releases/mgmt-$(VERSION)-release.url` at the end so the file is deleted if there's an error. (Or something similar)

And if the `hub release create` command echos anything useful, then you could tack on a `&& cat releases/mgmt-$(VERSION)-release.url` before the `||` so that we print it out to screen too. =D

Reply:
Done and tested. :D